skimage/transform/hough_transform.py

```diff
@@ -4,9 +4,10 @@
                                _hough_ellipse,
                                _hough_line,
                                _probabilistic_hough_line as _prob_hough_line)
+from .._shared._deprecators import kwonly_change
 
-
-def hough_line_peaks(hspace, angles, dists, min_distance=9, min_angle=10,
+@kwonly_change('1.0')
```

Comment:
Ironically, I think this should be a keyword-only argument. 😂 

```suggestion
@kwonly_change(until_version='1.0')
```